cob_cartesian_controller/src/trajectory_profile_generator/trajectory_profile_generator_ramp.cpp

```diff
@@ -0,0 +1,152 @@
+/*!
+ *****************************************************************
+ * \file
+ *
+ * \note
+ *   Copyright (c) 2015 \n
+ *   Fraunhofer Institute for Manufacturing Engineering
+ *   and Automation (IPA) \n\n
+ *
+ *****************************************************************
+ *
+ * \note
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_cartesian_controller
+ *
+ * \author
+ *   Author: Christoph Mark, email: christoph.mark@ipa.fraunhofer.de / christoph.mark@gmail.com
+ *
+ * \date Date of creation: December, 2015
+ *   This class calculates the timings for a ramp velocity profile with respect to the interpolation rate of the system. It also adjusts the velocity in case of
+ *   a bad velocity / path-length ratio, which leads to a longer path time.
+ * \brief
+ *
+ ****************************************************************/
+#include "ros/ros.h"
+#include "cob_cartesian_controller/trajectory_profile_generator/trajectory_profile_generator_ramp.h"
+/* BEGIN TrajectoryProfileRamp ********************************************************************************************/
+inline cob_cartesian_controller::ProfileTimings TrajectoryProfileRamp::getProfileTimings(double Se, double te, double accl, double vel, bool calcMaxTe)
```

Comment:
please change according to https://github.com/ipa320/cob_control/pull/74/files#r47380086